doc/README.rst

```diff
@@ -1,33 +1,19 @@
-######################
-Building Documentation
-######################
-
-Instructions are given below to build the documentation.  The recommended method is using Docker, 
-however, alternative instructions are provided in case using Docker in not an option.
-
-To develop the content of this repository, it is recommended to launch a `sphinx-autobuild`
-server as per the instructions below. Once started, point a web-browser at
-http://127.0.0.1:8000. If running the server within a VM, remember to configure
-port forwarding.
+####################
+Documentation Source
+####################
 
-You can now edit the .rst documentation, and your web-browser content will automatically
-update.
+This folder contains source files for the documentation. The sources do not render well in GitHub or an RST viewer.
+In addition, some information is not visible at all and some links will not be functional.
 
-************
-Using Docker
-************
+**********************
+Building Documentation
+**********************
 
 =============
 Prerequisites
 =============
 
-Install `Docker <https://www.docker.com/>`_.
-
-Pull the docker container:
-
-.. code-block:: console
-
-    $ docker pull ghcr.io/xmos/doc_builder:main
+Use the `xmosdoc tool <https://github.com/xmos/xmosdoc>`_ either via docker or install it into a pip environment.
```

Comment:
xmosdoc is private

Reply:
I see - this readme is for building the docs and is not included in the docs, I think that's fine